bagh/bagh.json

```diff
@@ -5,6 +5,14 @@
   "status": "beschikbaar",
   "version": "0.0.1",
   "crs": "EPSG:28992",
+  "versioning": {
+    "request_parameter": "versie",
```

Comment:
De  attributen zijn nu in snake_case, maar veel amsterdam- (en json) schemas gebruiken meestal camelCase.

Reply:
:heavy_check_mark:  ga ik dat aanpassen.